internal/cmd/freeze-api/convert.go

```diff
@@ -125,6 +121,7 @@ func createConverters(from, to string) error {
 		fromType := fromTypes[typeName]
 		toType, ok := toTypes[typeName]
 		if !ok {
+			fmt.Println(typeName)
```

Comment:
Can it be removed? Otherwise we should probably add more information about this error.

Reply:
Yes, it should be removed.